driver/bpf/plumbing_helpers.h

```diff
@@ -417,6 +423,34 @@ static __always_inline int bpf_test_bit(int nr, unsigned long *addr)
 	return 1UL & (_READ(addr[BIT_WORD(nr)]) >> (nr & (BITS_PER_LONG - 1)));
 }
 
+#ifdef CAPTURE_SCHED_PROC_FORK
+static __always_inline bool drop_syscall_child_events(void *ctx, enum ppm_event_type evt_type)
```

Comment:
This seems like a different fix from the PR one, right?
If yes, can we split this to another PR?

Reply:
Question here a bit on the how to continue best: I actually added this to ensure driver tests are running fine with my changes. So I can split the series of this PR into separate ones..but would then like to merge this one first other I would have to rebase and porting the read variants back and forth.

Reply:
I think it's okay to have it here, I will uniform this logic between all the other drivers (kmod and modern bpf vfork) in another PR

Reply:
Thanks!